spec/models/user_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
```

Comment:
- [ ] As required for the project, kindly ensure that you also wrote Integration tests for your views by using capybara gem and all of them should pass.